csharp/src/Google.Protobuf/Reflection/MessageDescriptor.cs

```diff
@@ -34,6 +34,10 @@
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
+#if DOTNET35
+// Needed for ReadOnlyDictionary, which does not exist in .NET 3.5
+using Google.Protobuf.Collections;
```

Comment:
Strangely I can already see the file in the master branch. How is this possible?
https://github.com/google/protobuf/blob/master/csharp/src/Google.Protobuf/Collections/ReadOnlyDictionary.cs